core/store/src/lib.rs

```diff
@@ -289,21 +289,38 @@ pub fn read_with_cache<'a, T: BorshDeserialize + 'a>(
     Ok(None)
 }
 
-#[derive(Default, Debug)]
+#[derive(Debug)]
 pub struct StoreConfig {
     /// Attempted writes to the DB will fail. Doesn't require a `LOCK` file.
     pub read_only: bool,
     /// Re-export storage layer statistics as prometheus metrics.
     /// Minor performance impact is expected.
     pub enable_statistics: bool,
+    /// Maximum number of store files being opened simultaneously.
+    pub max_open_files: i32,
+    /// Cache size for ColState column.
+    pub col_state_cache_size: usize,
+}
+
+impl Default for StoreConfig {
+    fn default() -> StoreConfig {
+        StoreConfig {
+            read_only: false,
+            enable_statistics: false,
+            max_open_files: 10 * 1000,
```

Comment:
avoid specifying this value here again -and use the one from config::default instead